<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupPanelBase</class>
 <widget class="QWidget" name="SetupPanelBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="banner_label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/geest-banner.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>**Geospatial Assessment of Women Employment and Business Opportunities in the Renewable Energy Sector**</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>To get started, we need to select a project folder. The contents of this folder will be managed by Geest. It will store the project file and the working analysis results.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="open_existing_project_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>üìÇ Open an existing project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="create_new_project_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>‚≠êÔ∏è Create a new project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="open_project_group">
     <property name="title">
      <string>Open an existing project folder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="previous_project_combo"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="dir_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="open_project_button">
        <property name="text">
         <string>üìÇ Open Project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="new_project_group">
     <property name="title">
      <string>Start a new project</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>The folder you select should be empty.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_project_directory_button">
        <property name="text">
         <string>üìÇ Create or select a project directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>If you do not have a boundary layer you can quickly add one using the button below:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="world_map_button">
        <property name="text">
         <string>Add World Layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>For new projects you need to select a boundary layer ...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsMapLayerComboBox" name="layer_combo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>... and the boundary name column:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsFieldComboBox" name="field_combo"/>
      </item>
      <item>
       <widget class="QCheckBox" name="use_boundary_crs">
        <property name="text">
         <string>Use Coordinate Reference System of your boundary layer</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="cell_size_spinbox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Analysis cell size (m)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>If needed, use the QGIS selection tool to **select your areas of interest** and then press continue to prepare the project. This may take a few minutes....</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prepare_project_button">
        <property name="text">
         <string>‚öôÔ∏è Prepare project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>113</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>This plugin is built with support from the Canada Clean Energy and Forest Climate Facility (CCEFCFy), the Geospatial Operational Support Team (GOST, DECSC) for the project Geospatial Assessment of Women Employment and Business Opportunities in the Renewable Energy Sector.   </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>open_existing_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>open_project_group</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_existing_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>new_project_group</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>create_new_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>open_project_group</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>create_new_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>new_project_group</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_existing_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>open_existing_project_button</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>create_new_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>create_new_project_button</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>create_new_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>open_existing_project_button</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_existing_project_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>create_new_project_button</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
